src/mlpack/methods/random_forest/extra_trees_main.cpp

```diff
@@ -0,0 +1,278 @@
+    "into each leaf for it to be split. The " +
+    PRINT_PARAM_STRING("minimum_gain_split ") +
+    "parameter controls the minimum required gain for a decision tree to be"
+    "split.Larger values will force higher-confidence split_size. The " +
+    PRINT_PARAM_STRING("maximum_depth") +
+    "parameter specifies the maximum depth of a tree.  If " +
+    PRINT_PARAM_STRING("print_training_accuracy") + " is specified, the "
+                                                    "calculated accuracy on the training set will be printed."
+                                                    "\n\n"
+                                                    "Test data may be specified with the " +
+    PRINT_PARAM_STRING("test") + " "
+                                 "parameter, and if performance measures are desired for that test set, "
+                                 "labels for the test points may be specified with the " +
+    PRINT_PARAM_STRING("test_labels") + " parameter.  Predictions for each "
+                                        "test point may be saved via the " +
+    PRINT_PARAM_STRING("predictions") +
+    "output parameter.  Class probabilities for each prediction may be saved "
+    "with the " +
+    PRINT_PARAM_STRING("probabilities") + " output parameter.");
+// Example.
+BINDING_EXAMPLE(
+    "For example, to train a extra trees algorithm with a minimum leaf size of"
+    "20 using 10 trees on the dataset contained in " +
+    PRINT_DATASET("data") +
+    "with labels " + PRINT_DATASET("labels") + ", saving the output random "
+                                               "forest to " +
+    PRINT_MODEL("et_model") + " and printing the training "
+                              "error, one could call"
+                              "\n\n" +
+    PRINT_CALL("extra_trees", "training", "data", "labels", "labels",
+               "minimum_leaf_size", 20, "num_trees", 10, "output_model",
+               "rf_model","print_training_accuracy", true) +
+    "\n\n"
+    "Then, to use that model to classify points in " +
+    PRINT_DATASET("test_set") + " and print the test error given the labels" +
+    PRINT_DATASET("test_labels") + " using that model, while saving the "
+                                   "predictions for each point to " +
+    PRINT_DATASET("predictions") + ", one "
+                                   "could call "
+                                   "\n\n" +
+
+    PRINT_CALL("extra_trees", "input_model", "rf_model", "test", "test_set",
+
+    PRINT_CALL("extra_trees", "input_model", "et_model", "test", "test_set",
+               "test_labels", "test_labels", "predictions", "predictions"));
+BINDING_SEE_ALSO("@decision_tree", "#decision_tree");
+BINDING_SEE_ALSO("@hoeffding_tree", "#hoeffding_tree");
+BINDING_SEE_ALSO("@softmax_regression", "#softmax_regression");
+BINDING_SEE_ALSO("Extra Trees on Wikipedia",
+                 "https://en.wikipedia.org/wiki/Random_forest#ExtraTrees");
```

Comment:
I think it would be useful to add `@random_forest` and a link to the "Extremely Randomized Trees" paper to the see alsos here. :+1: